### ISSUE TITLE
Test case: Drug-drug

### ISSUE THREAD

**Issue by jkiddo** [Gravitate-Health/focusing-interfaces#3]

**Comment by jkiddo** [comment]
@cander2 do we have data in the examples that can support this test case?

**Comment by cander2** [comment]
Yes for Skilarence and Suliqua. Haven't added it to Karvea yet

**Comment by cander2** [comment]
Yes, we have this in the cava (formerly Karvea) example. Interaction with methotrexate

http://build.fhir.org/ig/HL7/vulcan-eproduct-info/ClinicalUseDefinition-e60575aa-e7b3-4461-a456-d310514f4041.html
